dts/bindings/ethernet/microchip,ksz8081.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip KSZ8081 Ethernet PHY device
+
+compatible: "microchip,ksz8081"
+
+include: ethernet-phy.yaml
+
+properties:
+  mc,reset-gpio:
```

Comment:
Not sure about the `mc,` prefixes as these are common properties for all devices?